oem/supermicro/manager.go

```diff
@@ -0,0 +1,52 @@
+//
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+package supermicro
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/stmcginnis/gofish/common"
+	"github.com/stmcginnis/gofish/redfish"
+)
+
+type Manager struct {
+	redfish.Manager
+	Oem ManagerOem
+}
+
+type ManagerOem struct {
+	Supermicro struct {
+		OdataType string `json:"@odata.type"`
+		NTP       common.Link
```

Comment:
Then make these fields lower case. The links aren't needed by the user of the library, just the methods being added below to access the objects that those links point to.

So combined with the above, it would just be something like:

```go
smManager, err := supermicro.FromManager(redfishManager)
if err != nil {
    // Handle it
}

ntp, err := smManager.NTP()
```

Reply:
You need them when unmarshalling, but not after that. See elsewhere in the library where the unmarshalling takes care of accessing internal details, then stores them as non-public fields that are used in a more user-friendly interface.